postboxes/views.py

```diff
@@ -0,0 +1,36 @@
+from django.views           import View
+from django.http            import JsonResponse
+from django.core.paginator  import Paginator, InvalidPage
+
+from postboxes.models       import Postbox, Letter
+
+class CollectionView(View):
+    def get(self, request, postbox_uuid):
+        try:
+            postbox = Postbox.objects.filter(uuid=postbox_uuid).first()
+
+            if postbox is None:
+                return JsonResponse({'message':'Invalid_uuid'}, status=400)
+
+            postboxes   = Letter.objects.filter(postbox=postbox)
```

Comment:
아래 데이터를 보면 `Letter` 정보는 사용하는 곳이 없는데, `postboxes   = Letter.objects.filter(postbox=postbox)`가 필요한가요?

그리고 페이지네이션은 request로 받아서 유연하게 호출할 수 있도록 수정해주세요!


```suggestion
            offset = request.GET...
            limit = request.GET...
            
            postboxs = Postbox.objects.filter(uuid=postbox_uuid)
            
            if not postbox.exists():
                return ..
            
            result = [{
                'id'        : postbox.id,
                'nickname'  : postbox.nickname,
                'image_url' : postbox.image_url,
                'caption'   : postbox.caption
            }for postbox in postboxs]


```